src/main/java/org/alliancegenome/curation_api/jobs/executors/LoadFileExecutor.java

```diff
@@ -220,7 +221,11 @@
 					// e.printStackTrace();
 					history.incrementFailed();
 					addException(history, e.getData());
-				} catch (Exception e) {
+				} catch (KnownIssueValidationException e) { 
+					Log.debug(e.getMessage());
+					history.incrementSkipped();
+				}
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.blocks.RightCurlyCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'}' at column 5 should be on the same line as the next part of a multi-block statement (one that directly contains multiple blocks: if/else-if/else, do/while or try/catch/finally).
<!-- __reviewdog__:ChBmYWQ0ZTUzMWMwN2YzMjU5EgpjaGVja3N0eWxl -->